src/main/clojure/conexp/math/util.clj

```diff
@@ -76,6 +76,22 @@
                 (apply f (advance-coord-by args n (- precision))))
              (* 2 precision)))))))
 
+
+;;; Combinatorics
+(defn binomial-coefficient
+   "Straight forward computing of ${n\choose k}$ by multiplying
+  all (n-k) factors, where `run' is incremented up to k, divided by
+  the incrementing divisor `run'"
+  [n k]
+  (assert (>= n k))
```

Comment:
This assertion should not be necessary; the binomial coefficient is zero if n < k